### ISSUE TITLE
Remove unused code from "item-counter" calydon instance

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#6712]
Since the item counter implementation has been merged (coming from 2.0 branch) and have taken a different journey for a11y. The code below is no longer used:

`projects/storefrontstyles/scss/themes/calydon/components/cart/_item-counter.scss`

![Screen Shot 2020-02-25 at 12 06 22 PM](https://user-images.githubusercontent.com/15113219/75269654-42db5400-57c7-11ea-868f-643bf7aad83b.png)

**Comment by KateChuen** [comment]
Tested on dev10. 
Missing the visual focus on item-counter on pdp and add-to-cart modal.
Also cannot navigate to it using a keyboard.

**Comment by developpeurweb** [comment]
@KateChuen 
On a strict adherence to Accessibility Guidelines: **This OK**, because:

1. Provides 2 accessible alternatives for keyboard users.
2. Focus goes directly to the input field to do the change manually without passing through the buttons.

But some improvement are suggested for the next Accessibility milestone in this other ticket GH-6879 please see full answer there.
